tools/notify-mattermost

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Did you change anything here except extracting this?

Reply:
adding the option to pass a message to the command, but beside of this it is the same.